app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/fragment/NavigationFragment.java

```diff
@@ -44,6 +52,11 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
   @Override
   public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
     super.onViewCreated(view, savedInstanceState);
+    if (wasNavigationStopped()) {
```

Comment:
For readability when the `Fragment` is being created, this can be extracted into a `private` method